packages/react-native-bpk-component-rating/src/ios/RatingBridge/RCTBPKRating.m

```diff
@@ -0,0 +1,92 @@
+/*
+* Backpack - Skyscanner's Design System
+*
+* Copyright 2016-2019 Skyscanner Ltd
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#import "RCTBPKRating.h"
+
+#import <React/RCTBridge.h>
+#import <React/RCTUIManager.h>
+
+@interface RCTBPKRating()
+@property(nonatomic, weak, nullable) RCTBridge *bridge;
+@end
+
+@implementation RCTBPKRating
+
+- (instancetype)initWithBridge:(RCTBridge *)bridge {
+    self = [super initWithFrame:CGRectZero];
+
+    if (self) {
+        self.bridge = bridge;
+    }
+
+    return self;
+}
+
+- (void)setNeedsLayout {
+    [super setNeedsLayout];
+
+    CGSize size = [self systemLayoutSizeFittingSize:UILayoutFittingCompressedSize];
```

Comment:
I dunno how thread safe this is? I think you are only meant to call this on a UI thread, but I think `setNeedsLayout` can be called on other threads